controls/radtreeview/how-to/wpf-windowsexplorer-like-treeview.md

```diff
@@ -34,229 +34,232 @@ This tutorial will combine in itself the following:
 
 1. Here is a simple __RadTreeView__ declaration:
 
-	#### __XAML__
+	#### __[XAML] Example 1: Declare RadTreeView__
 
 	{{region radtreeview-how-to-wpf-windowsexplorer-like-treeview_0}}
-		<telerik:RadTreeView x:Name="radTreeView" Margin="8">
-			<telerik:RadTreeViewItem Header="MyComputer"
-				DefaultImageSrc="/Images/MyComputer.png"/>
-		</telerik:RadTreeView>
-		{{endregion}}
+		<telerik:RadTreeView x:Name="radTreeView" Margin="8"/>
+	{{endregion}}
 
 	![](images/RadTreeView_HowToWindowsExplorerTree_010_WPF.PNG)
 
 	The treeview has an initial item with Header "MyComputer". When the user expands the "MyComputer" node, then all available drives will be displayed. When the user expands any drive, then all sub folders and files will be displayed. The same should happen when the user expands any sub directory.

```

Comment:
Since the MyComputer RadTreeViewItem is removed from the snippet above the picture and this paragraph are not correct. We can either add the RadTreeViewItem again, or remove this paragraph and the picture.